testing/src/package.json

```diff
@@ -15,7 +15,8 @@
     "generate-report": "marge cypress/reports/mochareports/report.json -f report -o cypress/reports/mochareports",
     "posttest": "npm run combine-reports && npm run generate-report",
     "test": "npm run scripts || npm run posttest",
-    "Run headless for all modes": "npm run test"
+    "Run headless for all modes": "npm run test",
+    "uploadResultTQ" : "npx ts-code uploadResult.ts"
```

Comment:
I believe that we do not need the npx here. It can be executed in the same way that we do for the _setup:db_: 
```
setup:db": "ts-node scripts/setupDB
```